src/components/Footer/Footer.js

```diff
@@ -1,7 +1,210 @@
+              <AiFillInstagram className="icon insta" />
+            </a>
+            <a href="https://www.youtube.com" target="_blank" rel="noreferrer">
+              <AiFillYoutube className="icon youtube" />
+            </a>
+            <a href="https://www.kakaocorp.com/page/" target="_blank" rel="noreferrer">
+              <RiKakaoTalkFill className="icon kakao" />
+            </a>
+            <a href="https://www.facebook.com" target="_blank" rel="noreferrer">
+              <FaFacebookF className="icon facebook" />
+            </a>
+            <a href="https://www.naver.com" target="_blank" rel="noreferrer">
+              <SiNaver className="icon naver" />
+            </a>
+          </TopImgBox>
+        </FooterTop>
+
+        <FooterCenter>
+          <CenterItem>
+            <span>이용약관</span>
+            <span>계정문의</span>
+            <span>입점문의</span>
+          </CenterItem>
+        </FooterCenter>
+        <FooterBot>
+          <Flex>
+            <FlexItemOne>
+              <p>대표: 이재욱</p>
+              <p>사업자등록번호 : 231-85-03914</p>
+              <p className="underline">사업자확인</p>
+              <p>통신판매 신고번호: 2021-서울서초-2084</p>
+            </FlexItemOne>
+          </Flex>
+          <Flex>
+            <FlexItemTwo>
+              <p>주소 : 경기도 용인시 수지구 상현동 508 광교우미뉴브 A동 525-3호</p>
+              <p>정보보호 책임자 : 백종호</p>
+              <p>쉘위 주식회사 대표 전화 : 070-5014-1282</p>
+              <p>이메일 : info@sooldamhwa.com</p>
+            </FlexItemTwo>
+          </Flex>
+          <NicePay>
+            <p>
+              술담화는 통신판매중개자로서 통신판매 당사자가 아니며, 판매자가 등록한 상품정보 및
+              거래에 대해 술담화는 책임을 지지 않습니다. 고객님의 안전거래를 위해 현금 등으로 결제
+              시 저희 쇼핑몰에서 가입한 NICE구매안전 (에스크로) 서비스를 이용하실 수 있습니다.
+            </p>
+          </NicePay>
+          <NicePay>
+            <p className="underline">서비스가입사실확인</p>
+          </NicePay>
+        </FooterBot>
+      </FooterWrapper>
+    </FooterContainer>
+  );
+};
 
 export default Footer;
+
+const FooterContainer = styled.div`
+  width: 100vw;
+  height: 30vh;
+  padding: 30px;
+`;
+
+const FooterTop = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const FooterWrapper = styled.div`
+  width: 60%;
+  margin: auto;
+
+  ${FooterTop} {
+    margin-bottom: 50px;
+  }
+`;
+
+const TopText = styled.div`
+  width: 30%;
+
+  .medium-text {
+    font-size: 16px;
+    font-weight: bold;
+    margin-bottom: 15px;
+  }
+
+  .small-text {
+    font-size: 12px;
+    color: #3e3e3e;
+  }
+`;
+const TopImgBox = styled.div`
+  width: 30%;
+
+  .icon {
+    width: 25px;
+    height: 25px;
+    padding: 4px;
+    background-color: #eee;
+    border-radius: 100%;
+    margin-right: 10px;
+    margin-bottom: 40px;
+
+    box-shadow: 3px 3px 8px 0px rgba(0, 0, 0, 0.2);
+    transition: all 0.3s;
+  }
+
+  .icon:hover {
+    background-color: #fcf1d8;
+    transform: rotate(15deg) scale(1.2);
+  }
```

Comment:
```suggestion
  .icon {
    width: 25px;
    height: 25px;
    padding: 4px;
    background-color: #eee;
    border-radius: 100%;
    margin-right: 10px;
    margin-bottom: 40px;

    box-shadow: 3px 3px 8px 0px rgba(0, 0, 0, 0.2);
    transition: all 0.3s;

    &:hover {
      background-color: #fcf1d8;
      transform: rotate(15deg) scale(1.2);
    }
  }
```

scss 문법이 모두 사용 가능하니 이런 식으로도 작성이 가능하겠네요.

Reply:
오오 ..! 감사합니다 네스팅하였습니다 !